### DISCUSSION TITLE
Allow swapping global CSS ("microsites")

### DISCUSSION THREAD

**Discussion by akx** [vercel/next.js#17776] [Ideas]
It would be super useful to be able to somehow select which global stylesheets are enabled, to allow for setting up microsites with entirely different design (basically CSS).

The most dynamic would probably be allowing to do this in a custom App component based on pageProps etc., but to begin with, a mapping (regexps to a list of CSS assets?) in the next.config.js file would probably be enough.

Cc @olpeh since we talked about this sort of thing on Koodiklinikka :)

**Comment by vikasg603** [comment]
The same issue, Don't know when NextJS will allow this.

**Comment by vikasg603** [comment]
Hey @timneutkens, @Timer, is there any way of doing this?
This is the widespread case scenario I am looking from the last 2 months.

We need this as I wanted multiple templates but cannot do it because all templates have different global CSS requirements.
